examples/devpub/Makefile

```diff
@@ -0,0 +1,45 @@
+PKGDIR=./pkg
+BASENAME=devpub
+VERSION=`grep VERSION version.go | cut -f2 -d\"`
+OS_LIST=linux darwin
+ARCH=amd64
+UNFORMATTED=$(shell gofmt -l *.go)
+
+all: build
+
+setup:
+	go get github.com/mitchellh/gox
+	go get github.com/tcnksm/ghr
+
+checksetup:
+	go get golang.org/x/tools/cmd/goimports
+
+check: checkfmt
+	go vet *.go
+	goimports -l *.go
+
+checkfmt:
+ifneq ($(UNFORMATTED),)
+	@echo $(UNFORMATTED)
+	exit 1
+else
+	@echo "gofmt -l *.go OK"
+endif
+
+build:
+	mkdir -p ${PKGDIR}
+	for x in "$(OS_LIST)" ; do \
+		gox -output="${PKGDIR}/{{.Dir}}_{{.OS}}_{{.Arch}}" -os="$$x" -arch="${ARCH}" ; \
+	done
+
+release: build
+	ghr -u groovenauts -r blocks-concurrent-subscriber --replace --draft ${VERSION} pkg
+
+prerelease: build
+	ghr -u groovenauts -r blocks-concurrent-subscriber --replace --draft --prerelease ${VERSION} pkg
```

Comment:
@minimum2scp Thanks! I removed them. 
I was considering to make another repository for `devpub` go application. But I'm going to put it off.